### DISCUSSION TITLE
Removing Quadruple Points

### DISCUSSION THREAD

**Discussion by syedauc** [mtex-toolbox/mtex#772] [Ask Anything]
Hi, 
I am using MTEX 5.0.3 version. When I tried to use the following code to remove the quadruple points it did nit work. The quadruples are still in the orientation map and no error message was given. Can anyone help me on this? Thanks in advance

Syeda

[grains,ebsd.grainId] = calcGrains(ebsd,'angle',5*degree, 'removeQuadruplePoints')

**Comment by kilir** [comment]
Hi,

this option isn't so old, so updating to a recent version of mtex should help.

Cheers,
Rüdiger
